receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_windows.go

```diff
@@ -140,7 +140,7 @@ func stopSampling(_ context.Context) error {
 		// no load scraper is running nothing to do
 		return nil
 	}
-	
+
```

Comment:
nit: This is unrelated - can it be removed from this PR? (Seems a reasonable change to submit separately though)